plugin/vifm.vim

```diff
@@ -122,9 +119,6 @@ function! s:StartVifm(mods, count, editcmd, ...)
 			function! data.on_exit(id, code, event)
 				buffer #
 				silent! bdelete! #
-				if self.split
-					close
-				endif
```

Comment:
Without these three lines in neovim (but not in Vim) I get split window left on the screen, so one more window is created after each `:Vifm` with `:let g:vifm_embed_split = 1`.